products/BellHybrid/apps/application-bell-background-sounds/include/application-bell-background-sounds/ApplicationBellBackgroundSounds.hpp

```diff
@@ -20,6 +21,9 @@ namespace app
     }
     inline constexpr auto applicationBellBackgroundSoundsName = "ApplicationBellBackgroundSounds";
 
+    inline auto backgroundSoundsFilesPath =
```

Comment:
Is it really required to put it here? Does its scope need to be global? I see that it is only used in `void ApplicationBellBackgroundSounds::createUserInterface()`.

Reply:
maybe it should be passed as application parameter.
We should provide similar appraoch for music player on pure

Reply:
Moved it to cpp file and anonymous namespace. I'm not sure about passing it as application parameter. It can be discussed, however as for readability, storing it properly in application may be better or at least similair. Please continue the discussion on Monday if necessary.